facebook_connector.py

```diff
@@ -0,0 +1,40 @@
+from fbchat_archive_parser.parser import MessageHtmlParser
+import fnmatch
+import os
+import pandas as pd
+
+
+def initialize(dump_location='.'):
+    global messages_file
+    fb_message_filename = 'messages.htm'
+    messages_file = os.path.join(dump_location, fb_message_filename)
+    if not os.path.isfile(messages_file):
```

Comment:
I'd check if the file is here, and check one level deeper in case the user is pointing to the root of their fb dump.  Also the error message could be better, instead of telling them to move files, ask them to give you the location of the dump